packages/sanity/src/templates/resolve.ts

```diff
@@ -1,19 +1,19 @@
 import {
-  Schema,
   ArraySchemaType,
-  // InitialValueParams,
+  InitialValueProperty,
```

Comment:
This file is the most critical spot; many codepaths all of which needs context drilled.